content/tutorials/ios-push-tutorial-server-registration.textile

```diff
@@ -0,0 +1,64 @@
+
+In this tutorial, we'll see how you can setup and send push notifications to iOS your app using the "Ably Push Notification service":/general/push. You will also learn how to test this integration by sending notifications to an iOS device.
+
+Ably supports two types of client permissions for access to the Push Notification service:
+
+* @push-subscribe@: A client with this permission is a push target device, and it can manage its registration and any subscription for itself. However, it won't be able to manage push registrations or channel subscriptions for any other device. 
+* @push-admin@: A client with this contain permission has full access to the "Push admin API":https://www.ably.io/documentation/general/push/admin, and can manage registrations and subscriptions for all devices.
+
+Read more about permissions (also called capabilities) in our "API documentation":/core-features/authentication#capabilities-explained.
+
+By default, the Push Notification service provides the @AblyRealtime.push.activate@ method for the device to register itself with Ably directly, but you can instead delegate the device registration and subscription to your server. In this tutorial we will look at registration via server. You can jump to "another tutorial":/tutorials/ios-push-tutorial-direct if you'd rather like to have the device directly register itself with Ably.
+
+<%= partial 'tutorials/_step-1-setup-free-account' %>
+
+h2(#step2). Step 2 - Creating P12 Certificate
+
+<%= partial 'shared/tutorials/_create-p12' %>
+
+h2(#step3). Step 3 - Setting up Apple Push Notifications
+
+<%= partial 'shared/tutorials/_setup-apns' %>
+
+h2(#step4). Step 4 - Adding push to your application
+
+<%= partial 'shared/tutorials/_setup-push-ios-server-registration' %>
+
+<a href="https://github.com/ably/tutorials/pull/54/commits/0dc5cd43825af9f8f329a029ce27b34955f5c2ac">See this step in Github</a>
+
+h2(#step5). Step 5 - Setting up a Node.js server for authentication
+
+<%= partial 'shared/tutorials/_setup-node-push-auth' %>
+
+<a href="https://github.com/ably/tutorials/pull/54/commits/b35f42cc57c6eab060f9a8be964c92d163c7e533">See this step in Github</a>
+
+h2(#step6). Step 6 - Setting up device registration via the server
+
+<%= partial 'shared/tutorials/_setup-node-push-registration' %>
+
+<a href="https://github.com/ably/tutorials/pull/54/commits/50f146a0c6f8fa10a0b00abc6391454c2119e938">See this step in Github</a>
+
+h2(#step7). Step 7 - Using ngrok to make server public
+
+<%= partial 'shared/tutorials/_setup-ngrok' %>
+
+h2(#step8). Step 8 - Testing the app
+
+<%= partial 'shared/tutorials/_testing_push_ios' %>
+
+<a href="https://github.com/ably/tutorials/pull/54/commits/fecb121e4b66cbb6e3a8ce632f393f18179bf328">See this step in Github</a>
+
+h2(#furtherReading). Further Reading
```

Comment:
@cjbatin please check the brief document. It has specific details of what you need to include in this section. Please update it with the info.